book/src/install_config.md

```diff
@@ -13,6 +13,8 @@ cat << EOF > ~/.config/prr/config.toml
 [prr]
 token = "$YOUR_PAT_FROM_LAST_STEP"
 workdir = "/home/dxu/dev/review"
+
+activate_pr_metadata_experiment = false
```

Comment:
Might be good to drop this from install guide. B/c if/when we do change/remove this flag,
we might unnecessarily break a bunch of people who were blindly copy pasting this config.

At some point we will stabilize this feature, so users will be getting it anyways. (if
you were worried about the feature not getting enough exposure)